src/pageObject.js

```diff
@@ -1,15 +1,38 @@
 'use strict';
 
 class Header {
-
+  logo = 'Conduit logo';
 };
 
-class PageObject {
+class PageObject extends Header {
+  header = new Header();
+  constructor(url) {
+    super();
+    this.url = url;
+  }
 
+  clickOnLogo() {
+    return `Click on the ${this.logo}`;
```

Comment:
Replace `this.logo` with `this.header.logo` to access the logo from the Header instance.